### DISCUSSION TITLE
wifi_img_streamer_with_ap.bin, it hangs at 6%

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#442] [Q&A]
I noticed that it's a different issue, so I create a new discussion. 
I reinstalled the firmware 2022.05, and I also had a not-taking-off problem. Then, I installed the firmware 2022.09 and flashed the gap8 bootloader. After that, I tried to flash the wifi_img_streamer_with_ap.bin, but it hangs at 6%... I have already tried many times on different days.

**Comment by knmcguire** [comment]
Hi!

That sounds bad indeed and it seems that the reflashing of the bootloader did not go well. Could you try that again perhaps?

Also, I'm interested, as your gap8 bootloader was working fine before, why did you think it was necessary to reflash it with a programmer? [The instructions](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader-for-the-aideck-1-0-or-aideck-1-1-rev-a-c-) shows this is only necessary on older versions of the AIdeck, and if you were flashing the wifi example okay before, then this should not been necessary (and actually harmful apparently). Should we put a bigger warning at those instructions?

**Reply by noahzn** [reply]
Thanks for your reply. My AIdeck is Rev C1. Since there were errors running the fpv script, I just wanted to re-install it from the beginning of the tutorial. Is there a way to solve this?

**Reply by knmcguire** [reply]
Could you just try building flashing the gap8 bootloader again ? perhaps a couple of times before it works.
